google-cloud-spanner/test/google/cloud/spanner/database/update_test.rb

```diff
@@ -85,4 +88,50 @@
     _(job).must_be_kind_of Google::Cloud::Spanner::Database::Job
     _(job).wont_be :done?
   end
+
+  it "updates with PROTO BUNDLE and file descriptor set" do
+    proto_string = <<~PROTO
+      syntax = "proto3";
+      package examples;
+
+      message Foo {
+        string bar = 1;
+      } 
+    PROTO
+
+    descriptor_set = parse_descriptor_from_proto_string(proto_string)
+    encoded_data = Base64.encode64(Google::Protobuf::FileDescriptorSet.encode(descriptor_set))
+
+    ddl_proto_statement = <<~CREATE_PROTO
+        CREATE PROTO BUNDLE (
+          examples.Foo
+        )
+      CREATE_PROTO
+
+    ddl_table_statement = <<~CREATE_TABLE
+        CREATE TABLE Foos (
+          Id INT64 NOT NULL,
+          Foo `examples.Foo` NOT NULL, 
+        )
+      CREATE_TABLE
+
+    update_res = \
```

Comment:
You don't need the backslash, because Ruby knows the assignment operator implies that something will be on the next line. (Rubocop should be flagging this.)